app/tasks/FetchContributorsTask.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Website\Cli\Tasks;
 
+use const APP_PATH;
```

Comment:
`APP_PATH` is the path of the application set in `index.php`. This use statement is the new "auto import" from PHP7 that speeds things up. Again PHPStorm does that automatically :)